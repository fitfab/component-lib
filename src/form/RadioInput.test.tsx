import { cleanup, render } from '@testing-library/react'
import React from 'react'
import { RadioInput } from './RadioInput'

afterEach(cleanup)

test('Render RadioInput', () => {
    const { container } = render(<RadioInput name="one" />)
    expect(container).toMatchSnapshot()
})
