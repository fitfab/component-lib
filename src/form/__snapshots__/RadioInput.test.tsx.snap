// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render RadioInput 1`] = `
<div>
  <input
    class="sc-bdVaJa bSYQVI"
    name="one"
    type="radio"
    value=""
  />
  <label />
</div>
`;
